convert2rhel/unit_tests/utils_test.py

```diff
@@ -460,3 +460,27 @@ def test_remove_orphan_folders(path_exists, list_dir, expected, tmpdir, monkeypa
 
     utils.remove_orphan_folders()
     assert os_remove_mock.call_count == expected
+
+
+class DummyPopenOutput(unit_tests.MockFunction):
+    def __init__(self, output):
+        self.output = output
+
+    def __call__(self, args, stdout, stderr, bufsize):
+        return self
+
+    @property
+    def stdout(self):
+        return self
+
+    def readline(self):
+        return self.output.split()
```

Comment:
`split()` splits a string by a whitespace character. So splitting the string you're passing to it "test of nonascii output: cafe" gives you a list of words. The `readline()` function should return a string though.

Also, the stdout is supposed to be a binary stream. So you should convert the string to a binary stream.

Reply:
Thank you for pointing this out I have made the adjustments to this and will push it!